crates/rpc-types-beacon/src/lib.rs

```diff
@@ -2,12 +2,13 @@
 //!
 //! Provides all relevant types for the various RPC endpoints, grouped by namespace.
```

Comment:
seems this should be updated to say this is beacon-types-specific

Reply:
Updated in README